app/components/helix/buttons/Button.js

```diff
@@ -1,20 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Button as HelixButton } from '@helix-design-system/helix-react';
+import classnames from 'classnames';
 
 class Button extends React.Component {
   render() {
-    const { label, disabled, classNames, onClick, processing, submit, children } = this.props;
+    const {
+      label,
+      disabled,
+      classNames,
+      onClick,
+      processing,
+      submit,
+      children,
+      variant,
+      size,
+      wide
+    } = this.props;
+    const buttonClassNames = classnames(classNames, {
+      'submit-btn': submit,
+      'btn-wide': wide
+    });
     return (
-      <button // eslint-disable-line react/button-has-type
-        className={`hxBtn ${classNames}`}
+      <HelixButton
+        className={buttonClassNames} // helix automatically adds a space between prefilled and added classes
         type={submit ? 'submit' : 'button'}
         disabled={disabled}
         onClick={onClick}
+        variant={variant}
+        size={size}
+        busy={processing}
       >
         {!processing && (label || children)}
         {processing && <span>{label || children}</span>}
```

Comment:
non blocking: ternary might be a little more clear
```suggestion
        {processing ? <span>{label || children}</span> : (label || children)}
```